src/include/miopen/solver.hpp

```diff
@@ -108,6 +108,8 @@ struct SolverBase
     /// run-time parameters.
     virtual bool IsDynamic() const { return false; }
 
+    static constexpr float wti_unknown = -2;
```

Comment:
[notice] Introducing a symbol here logically leads us to introduce symbols in other similar places.